#ifndef __NORI_DICTIONARY_H__
#define __NORI_DICTIONARY_H__

#define NORI_DICT_FILE "dictionary.bin"
#define NORI_DICT_META_FILE "dictionary_meta.pb"
#define NORI_UNK_FILE "unk.pb"

namespace nori {
namespace dictionary {}  // namespace dictionary
}  // namespace nori

#endif  //__NORI_DICTIONARY_H__
